src/test/java/com/limechain/network/protocol/transaction/state/TransactionStateTest.java

```diff
@@ -0,0 +1,126 @@
+package com.limechain.network.protocol.transaction.state;
+
+import com.limechain.transaction.TransactionState;
+import com.limechain.transaction.dto.Extrinsic;
+import com.limechain.transaction.dto.TransactionValidity;
+import com.limechain.transaction.dto.ValidTransaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.Comparator;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class TransactionStateTest {
+
+    private static final byte[] TAG_1 = "tag1".getBytes();
+    private static final byte[] TAG_2 = "tag2".getBytes();
+
+    private TransactionState sut;
```

Comment:
can you rename this field

Reply:
https://en.wikipedia.org/wiki/System_under_test